app/Http/Controllers/API/RecordsController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use Request;
+
+class RecordsController extends Controller
+{
+    public function getRecords(){
```

Comment:
This function is not necessary for the task (it's covered by https://app.asana.com/0/943748898987942/943771407436716). If you want to work on that one too, after finishing this, you can keep the function and continue with the filter afterward. Otherwise, please remove it.